python/py-mpmath/Portfile

```diff
@@ -33,6 +33,8 @@ if {${name} ne ${subport}} {
         checksums           rmd160  7b0ea9200d2b5f9d6a139e8d5b67ecd07e967952 \
                             sha256  fc17abe05fbab3382b61a123c398508183406fa132e0223874578e20946499f6 \
                             size    512600
+    } else {
+        depends_build-append    port:py${python.version}-setuptools_scm
```

Comment:
it seems to me from [upstream's setup.py](https://github.com/fredrik-johansson/mpmath/blob/1.2.1/setup.py) that this also needs a build dependency on `py-setuptools`, correct?

Reply:
Indeed, though that was already required before the 1.2 update. Probably nobody ever noticed because pretty much every python Portfile depends on _py-setuptools_ and it's thus likely installed everywhere. It doesn't need to be explicitly added here because _py-setuptools_scm_ obviously depends on _py-setuptools_. I checked some other ports that depend on _py-setuptools_scm_ and only about half of them explicitly declare their dependency on _py-setuptools_, the other half only have the transitive dependency.

Reply:
true, sorry I missed that - this looks good then.